cvat-sdk/cvat_sdk/core/proxies/tasks.py

```diff
@@ -257,23 +257,30 @@ def export_dataset(
         pbar: Optional[ProgressReporter] = None,
```

Comment:
Probably, you need to merge the develop branch, as this function was changed.

Reply:
ahhh, this was my issue. I was doing my work off my older fork. I've gone ahead and added a few tests for the new functionality and changed the naming of the all_images parameter to only_annotated. Let me know if any more work is needed